src/pyscipopt/scip.pxi

```diff
@@ -5445,6 +5537,155 @@ cdef class Model:
         """Get an estimation of the final tree size """
         return SCIPgetTreesizeEstimation(self._scip)
 
+@dataclass
+class Statistics:
+    """
+    Attributes
+    ----------
+    total_time : float
+        Total time since model was created
+    solving_time: float
+        Time spent solving the problem
+    presolving_time: float
+        Time spent on presolving
+    reading_time: float
+        Time spent on reading
+    copying_time: float
+        Time spent on copying
+    problem_name: str
+        Name of problem
+    presolved_problem_name: str
+        Name of presolved problem
+    _variables: dict
+        Dictionary with number of variables by type 
+    _presolved_variables: dict
+        Dictionary with number of presolved variables by type 
+    _constraints: dict
+        Dictionary with number of constraints by type
+    _presolved_constraints: dict
```

Comment:
no need to add public documentation for non-public functions, these should probably be normal comments